Makefile

```diff
@@ -1,16 +1,17 @@
-.PHONY: all clean test lint install
+.PHONY: all clean test lint build install
 
 ROOT_DIR=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 SRCDIRS= sensorapp/sensorserver sensorapp/sensorfetcher camerapp/imageserver camerapp/imagefetcher bwtester/bwtestserver bwtester/bwtestclient bat tools/pathdb_dump ssh/client ssh/server netcat webapp _examples/helloworld _examples/shttp/client _examples/shttp/server
-TARGETS = $(foreach D,$(SRCDIRS),$(D)/$(notdir $(D)))
+BUILD_TARGETS = $(foreach D,$(SRCDIRS),$(D)/$(notdir $(D)))
+INTEGRATION_TARGETS = $(foreach D,$(SRCDIRS),$(D)/$(notdir $(D))_integration)
 
-all: lint $(TARGETS)
+all: lint $(BUILD_TARGETS)
 
 clean:
 	go clean ./...
 
 test: lint
-	go test ./...
+	go test -tags=unit ./...
```

Comment:
I found it less confusing having two separate tags for unit tests and integration tests, rather than having to negate an unrelated tag when running unit tests.
Changed it as suggested.

Reply:
Thanks